script2

```diff
@@ -0,0 +1,76 @@
+
+// const addButton = document.querySelector('.addButton');
+// const inputValue = document.querySelector('.input');
+// const container = document.querySelector('.container');
+// const itemList = [];
+
+// class Item {
+//   constructor(itemName) {
+//     this.itemName = itemName;
+//     this.createDiv(itemName);
+//     itemList.push(this.itemName);
+//     this.saveItems();
+//   }
+
+//   createDiv(itemName) {
+//     const input = document.createElement('input');
+//     input.value = itemName;
+//     input.disabled = true;
+//     input.classList.add('item_input');
+//     input.type = 'text';
+
+//     const itemBox = document.createElement('div');
+//     itemBox.classList.add('item');
+//     container.appendChild(itemBox);
+//     itemBox.appendChild(input);
+
+//     const editButton = document.createElement('button');
+//     editButton.textContent = 'Edit';
+//     editButton.classList.add('editButton');
+//     itemBox.appendChild(editButton);
```

Comment:
- Good job removing the commented code highlighted by the previous reviewer :clap: however after doing it you have commented out this part,kindly remove it to got to the next project :recycle: